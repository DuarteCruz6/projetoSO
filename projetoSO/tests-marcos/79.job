WRITE [(m9,15)(i5,53)(q2,48)(n7,48)(o4,89)]
WRITE [(d2,74)(h3,79)(d4,77)]
WRITE [(q6,37)(l8,2)(w7,34)(r8,57)]
WRITE [(s7,23)]
WRITE [(z9,76)]
DELETE [s7,h3]
WRITE [(g2,70)(h8,16)(f0,53)(m1,30)]
SHOW
READ [z9]
WRITE [(e9,47)]
WRITE [(x7,28)(t0,36)]
READ [o4]
WRITE [(u9,69)(d7,99)]
WRITE [(q8,51)(t1,39)(w2,95)(l2,22)(n1,23)]
WRITE [(k0,9)(q3,92)(v8,8)(a2,84)(e3,63)]
WRITE [(u4,94)(n6,16)(i1,21)(p0,23)]
WRITE [(x6,99)(t5,12)]
WRITE [(k4,42)(q1,83)(e7,23)(f1,33)]
DELETE [w2]
DELETE [k4]
WRITE [(g4,19)(j6,81)(i0,49)(i7,6)(u7,91)]
WRITE [(o8,22)(r5,56)(c7,74)]
WRITE [(z7,94)(k6,7)]
WRITE [(c6,24)(e1,90)(v6,17)(l6,79)]
WRITE [(h7,89)(a6,61)(a3,73)(s7,37)]
WRITE [(x5,45)(o2,57)(j0,79)]
DELETE [w7]
WRITE [(j4,36)(a5,6)(y2,79)(h5,42)(y1,45)]
READ [i0]
WRITE [(j3,5)(j3,36)(o5,54)]
DELETE [v8]
WRITE [(u2,19)]
WRITE [(o6,55)(y8,11)(c2,1)(r9,70)(k4,86)]
READ [c2]
WRITE [(x9,47)(k2,63)]
WRITE [(w3,70)(a8,5)(j1,80)]
WRITE [(w2,84)(s3,87)(d5,87)]
WRITE [(q5,95)(w4,18)(z8,90)(v1,38)]
WRITE [(i2,69)]
READ [x6]
READ [a5]
DELETE [x5,q3,t5,s7]
READ [d5]
WRITE [(g6,62)(o9,83)(v2,65)(e2,11)]
WRITE [(t9,42)(v8,52)]
READ [q2]
WRITE [(m5,82)(d0,46)(q3,32)]
WRITE [(f6,56)]
READ [v1]
READ [c2]
DELETE [t1,g2,v8]
WRITE [(d8,40)]
WRITE [(m2,61)(f8,43)]
WRITE [(c5,85)(l5,23)(y0,67)(d6,16)(c8,20)]
WRITE [(z2,96)]
DELETE [s3,o9]
READ [i5]
WRITE [(n0,39)(s8,52)(n8,67)(k8,43)]
WRITE [(q4,91)(b3,98)(b5,23)(m8,58)]
READ [z7]
SHOW
READ [l6]
WRITE [(b0,48)(t6,54)]
WAIT 1000
WRITE [(m0,75)(m4,0)(e8,9)(b2,50)]
WRITE [(p8,93)(z4,84)]
READ [i7]
SHOW
SHOW
WRITE [(p3,43)(t2,89)]
WRITE [(v8,62)(m6,39)]
WAIT 3000
READ [e9]
WRITE [(t1,37)]
DELETE [o8,c5]
WRITE [(p4,65)]
DELETE [m8,z4,q5,n6,q8]
WRITE [(c4,67)(a4,16)(c4,13)]
SHOW
WRITE [(t8,55)]
BACKUP
WRITE [(x2,3)(w7,42)(a7,5)]
BACKUP
WRITE [(i6,16)(l4,1)(z6,22)]
READ [z8]
WRITE [(o3,3)(z4,52)]
READ [p0]
READ [p0]
DELETE [u9,l2,x6,o5]
SHOW
WAIT 1000
WRITE [(w0,84)(c3,33)]
WRITE [(n3,17)]
WRITE [(u6,13)(y9,26)(i3,39)]
READ [n8]
WRITE [(j2,90)(j9,96)(g8,87)(n6,12)(g9,88)]
WRITE [(n9,28)(r4,40)]
