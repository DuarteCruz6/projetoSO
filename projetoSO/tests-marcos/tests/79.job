WRITE [(r9,70)(y9,48)(e8,91)(m4,16)]
READ [m4]
WRITE [(s4,98)]
READ [y9]
WRITE [(y1,66)(v3,83)(j7,64)(n9,46)(j1,59)]
DELETE [e8]
READ [y1]
DELETE [y9,y1,v3,m4]
DELETE [j1,n9,s4,r9]
DELETE [j7]
WRITE [(a6,86)(v4,67)]
WRITE [(r3,96)(q6,3)]
WRITE [(g9,56)(v1,96)(o6,50)]
WRITE [(p7,34)(w9,3)(p2,39)(h1,89)]
WRITE [(d0,19)(x4,31)]
WRITE [(c4,80)(m0,56)(d2,42)]
WAIT 3000
BACKUP
WRITE [(g6,32)]
WAIT 2000
WRITE [(f0,28)]
WRITE [(b7,18)(d9,0)(f1,51)(g8,34)(q2,27)]
READ [v1]
DELETE [g8,p7]
WRITE [(h4,44)]
DELETE [g9]
WRITE [(d4,70)(g9,39)(m9,18)(n6,100)(f7,12)]
WRITE [(f8,12)(d8,1)(z0,69)(x2,21)(m5,55)]
WRITE [(t0,8)(f6,23)(j8,3)]
DELETE [d2]
DELETE [g9]
DELETE [o6]
WRITE [(f5,81)(k5,93)(r1,4)]
WRITE [(h6,74)(g3,4)]
WRITE [(y9,57)]
READ [x2]
WRITE [(n9,18)(w4,5)(o0,63)(p7,90)(p8,24)]
SHOW
SHOW
READ [f0]
READ [h1]
DELETE [r1]
WRITE [(k1,51)]
WRITE [(e4,71)(c0,33)]
DELETE [f7,o0]
WAIT 2000
READ [k5]
WRITE [(q0,11)]
READ [r3]
BACKUP
WRITE [(e2,77)(g8,81)(b4,30)(h5,42)(l5,71)]
READ [q0]
WRITE [(r5,13)(r5,48)(r6,92)(p5,74)]
