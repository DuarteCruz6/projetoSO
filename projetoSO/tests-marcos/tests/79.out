[(m4,16)]
[(y9,48)]
[(y1,66)]
[(v1,96)]
[(x2,21)]
(a6, 86)
(b7, 18)
(c4, 80)
(d8, 1)
(d4, 70)
(d9, 0)
(d0, 19)
(f5, 81)
(f6, 23)
(f8, 12)
(f7, 12)
(f1, 51)
(f0, 28)
(g3, 4)
(g6, 32)
(h6, 74)
(h4, 44)
(h1, 89)
(j8, 3)
(k5, 93)
(m5, 55)
(m9, 18)
(m0, 56)
(n9, 18)
(n6, 100)
(o0, 63)
(p8, 24)
(p7, 90)
(p2, 39)
(q2, 27)
(q6, 3)
(r1, 4)
(r3, 96)
(t0, 8)
(v1, 96)
(v4, 67)
(w4, 5)
(w9, 3)
(x2, 21)
(x4, 31)
(y9, 57)
(z0, 69)
(a6, 86)
(b7, 18)
(c4, 80)
(d8, 1)
(d4, 70)
(d9, 0)
(d0, 19)
(f5, 81)
(f6, 23)
(f8, 12)
(f7, 12)
(f1, 51)
(f0, 28)
(g3, 4)
(g6, 32)
(h6, 74)
(h4, 44)
(h1, 89)
(j8, 3)
(k5, 93)
(m5, 55)
(m9, 18)
(m0, 56)
(n9, 18)
(n6, 100)
(o0, 63)
(p8, 24)
(p7, 90)
(p2, 39)
(q2, 27)
(q6, 3)
(r1, 4)
(r3, 96)
(t0, 8)
(v1, 96)
(v4, 67)
(w4, 5)
(w9, 3)
(x2, 21)
(x4, 31)
(y9, 57)
(z0, 69)
[(f0,28)]
[(h1,89)]
[(k5,93)]
[(r3,96)]
[(q0,11)]
